org.alloytools.alloy.classic.provider/src/main/java/org/alloytools/alloy/classic/provider/AlloyClassicFacade.java

```
@@ -0,0 +1,365 @@
+					return new AlloyModuleClassic() {
+
+						@Override
+						public Set<TSig> getSigs() {
+							ConstList<Sig> sigs = module.getAllReachableSigs();
+							return new HashSet<>(sigs);
+						}
+
+						@Override
+						public Optional<TSig> getSig(String name) {
+							return getSigs().stream()
+								.filter(s -> s.getName()
+									.equals(name))
+								.findAny();
+						}
+
+						@Override
+						public List<TRun> getRuns() {
+							Module THIS = this;
+							return module.getAllCommands()
+								.stream()
+								.filter(c -> !c.isCheck())
+								.map(r -> (TRun) new AbstractCommand(this, r))
+								.collect(Collectors.toList());
+						}
+
+						@Override
+						public List<TCheck> getChecks() {
+							Module THIS = this;
+							return module.getAllCommands()
+								.stream()
+								.filter(c -> c.isCheck())
+								.map(r -> (TCheck) new AbstractCommand(this, r))
+								.collect(Collectors.toList());
+						}
+
+						@Override
+						public CompModule getOriginalModule() {
+							return module;
+						}
+
+						@Override
+						public Optional<String> getPath() {
+							return Optional.ofNullable(path);
+						}
+
+						@Override
+						public List<CompilerMessage> getWarnings() {
+							return Collections.emptyList();
+						}
+
+						@Override
+						public List<CompilerMessage> getErrors() {
+							return Collections.emptyList();
+						}
+
+						@Override
+						public boolean isValid() {
+							return getErrors().isEmpty();
+						}
+
+						@Override
+						public String toString() {
+							return module.toString();
+						}
+
+						@Override
+						public Map<String, String> getSourceOptions(TCommand command) {
+							return extractOptions(options, command);
+						}
+
+						@Override
+						public Compiler getCompiler() {
+							return compiler();
+						}
+					};
+				} catch (Exception e) {
+					return new AlloyModuleClassic() {
```

I don't think we should just ignore this exception and return an empty module